distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestConfiguration.java

```diff
@@ -267,6 +272,18 @@ public void testExport() throws Exception {
     }
 
     @Test
+    public void testExportToDirectory() throws Exception {
+        resetInitialState();
+
+        String response = console.runCommand(EXPORT_COMMAND + " " + temporaryFolder.getRoot());
+
+        assertThat(String.format("Exporting current configurations to %s.",
```

Comment:
Shouldn't we also make sure that the files were actually exported like we do in `testExport()`?